### PR TITLE
Add mile time

### PR DESCRIPTION
**Pull request by GabrielGonzalezDiaz**
Added a mile time data point for creating a profile. Also fixed the name of the class of `DataHandler.java`

Please ignore the TODOs on DataHandler, since they are from changes that I have made on my branch, and cannot be seen on here.

### CODE REVIEW

File: capstone/fitness/src/main/java/com/google/sps/servlets/CreateProfileServlet.java [hunk 1]
```diff
@@ -56,6 +56,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Integer age = Integer.parseInt(request.getParameter(DataHandler.AGE_PROPERTY));
     Integer weeksTotrain = Integer.parseInt(request.getParameter(DataHandler.WEEKS_TO_TRAIN_PROPERTY));
     Float lengthOfMarathon = Float.parseFloat(request.getParameter(DataHandler.MARATHON_LENGTH_PROPERTY));
+    //Divide by 60 to convert to hours
```
Comment (sliang3) [inline capstone/fitness/src/main/java/com/google/sps/servlets/CreateProfileServlet.java:59]: nit: punctuation, also please add a space after //

File: capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java [hunk 2]
```diff
@@ -86,4 +94,6 @@ public static Entity getUser() {
     }
     return null;
   }
-}
\ No newline at end of file
+
+
```
Comment (sliang3) [inline capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java:98]: nit: remove extra lines.

File: capstone/fitness/src/main/webapp/profile-info.html [hunk 3]
```diff
@@ -41,6 +41,13 @@
         <br>
         <br>
 
+        <div class="input-container">
+          <label for="mileTime">How fast can you run you a mile in minutes (it's okay to estimate)</label>
+          <input class="separator" type="number" name="mileTime" min="1" step="0.01" required>
```
Comment (sliang3) [inline capstone/fitness/src/main/webapp/profile-info.html:46]: It's a little confusing that mileTime definition is different here compared to in the servlet where you're dividing by 60. Maybe you can rename this to minPerMile?

**Review by sliang3 — APPROVED** [review]

File: capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java [hunk 4]
```diff
@@ -13,11 +13,17 @@
...
+import java.util.Set;
 
 
-public class UserHelper {
+public class DataHandler {
 
   // Constants used for datastore.
+  // TODO(@gabrieldg) 
+  //  - make private and change to getters and setters.
+  //  - Figure out why the identifiers do not work.
```
Comment (lupengscu) [inline capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java:25]: What's not working?

File: capstone/fitness/src/main/webapp/profile-info.html [hunk 5]
```diff
@@ -41,6 +41,13 @@
         <br>
         <br>
 
+        <div class="input-container">
+          <label for="mileTime">How fast can you run you a mile in minutes (it's okay to estimate)</label>
```
Comment (lupengscu) [inline capstone/fitness/src/main/webapp/profile-info.html:45]: run a mile?

**Review by lupengscu — APPROVED** [review]

File: capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java [hunk 6]
```diff
@@ -13,11 +13,17 @@
...
+  private static Set<String> PROPERTIES  = new HashSet<>();
```
Comment (ike-elue) [inline capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java:26]: Whats the purpose of this HashSet?

**Review by ike-elue — APPROVED** [review]

File: capstone/fitness/src/main/webapp/profile-info.html (hunk 3 above)
Comment (GabrielGonzalezDiaz) [reply capstone/fitness/src/main/webapp/profile-info.html:46]: Well I was naming it trying to hint that it is the time in which the user can run a mile, and in data store everything is stored in hours, but asking the user for their time in hours for 1 mile seemed a bit strange.

File: capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java (hunk 4 above)
Comment (GabrielGonzalezDiaz) [reply capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java:25]: That comment is from something that was not included in this PR. Forgot to add to ignore that as well.

File: capstone/fitness/src/main/webapp/profile-info.html (hunk 5 above)
Comment (GabrielGonzalezDiaz) [reply capstone/fitness/src/main/webapp/profile-info.html:45]: Yes, for Pierce's feature he needs to know the time in which they can run one mile.

File: capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java (hunk 6 above)
Comment (GabrielGonzalezDiaz) [reply capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java:26]: That is for a different feature.

**Pull request merged**